package/common_init/overlay/etc/init.d/S99reset_boot_counter

```diff
@@ -42,6 +30,6 @@ if fw_printenv env_version_major; then
     fw_setenv post_boot_counter 0
   fi;
 else
-  logi --sbp "No uboot environment version found, intializing..."
+  logi --sbp "No uboot environment version found, not doing anything..."
```

Comment:
Can we remove the call to `initialize_uboot_environment` since it was deleted?  Also `$(initialize_uboot_environment)` is capturing and evaluating the output of `initialize_uboot_environment`, just `initialize_uboot_environment` is sufficient to call the function.